### ISSUE TITLE
orderingList: cannot mutate some widget options

### ISSUE THREAD

**Issue by jstefek** [richwidgets/richwidgets#80]
Some options cannot be set with:

```
orderingList('option','$optionName, $optionValue)
```

, because they are not implemented yet

List of these options:
- contained
- dragSelect
- dropOnEmpty
- mouseOrderable
- showButtons
- all events

**Comment by lfryc** [comment]
All the listed options either changes behavior of the drag-n-drop functionality or influences appearance of the widget (showButtons).

In those cases, it's not completely necessary to react for changes dynamically, since it's not common to change those options during widget's life - they are configured just once.

---

Implementing mutability of all options is nice to have, but I would say it's not necessary, at least not until `1.x`.

We should make sure to implement mutability for all options where it makes sense though.

**Comment by lfryc** [comment]
For the original report: events are mutable, when you change event using `$("#list").orderingList('option', 'change', function() { console.log('change2'); });` then the event is changed.

In my opinion `widgetEventPrefix` wouldn't have to be mutable at all, since as you instantiate the widget, you can register events that shouldn't change during the lifetime of the widget.

**Comment by lfryc** [comment]
@bleathem @jstefek let me know if you agree with my assessment. I'm pushing this issue to Future for now.

**Comment by bleathem** [comment]
Do we know if jQuery UI behaves the same?  Do jQuery UI widgets offer options that are not mutable via the "option" API?
